src/app/logo/logo.component.html

```diff
@@ -1,3 +1,3 @@
 <div class="app-logo-container">
-    STAR WARS API browser
+    STAR WARS APi browser
```

Comment:
In case you're wondering why 'i' is lowercase. I imported new font for logo to make it look like originial StarWars font and it just looks better that way.